src/handlers.js

```diff
@@ -132,10 +132,10 @@ class Handlers {
     }
     let body = [
       '<details>\n',
-      '<summary>Submitting the task to Taskcluster failed. Details</summary>\n\n',
-      '```js\n',
-      errorBody,
-      '```\n',
+      '<summary>Submitting the task to Taskcluster failed. Details</summary>',
+      '',
```

Comment:
Will that empty string render as a break between Summary element and the `errorBody`? Maybe a `<br>` in there instead? 

(If we need a break between them, that is)

Reply:
It will render as two newlines, which Markdown treats as a paragraph separator.